lms/djangoapps/certificates/models.py

```diff
@@ -466,6 +467,10 @@ def certificate_status_for_student(student, course_id):
                    should not be issued a certificate. This will
                    be set if allow_certificate is set to False in
                    the userprofile table
+    unverified   - The student was in verified enrollment track and
```

Comment:
I'll update the comment. Honor mode is a legacy course mode and going forward, we don't have to worry about that case. So I think we can remove that point here as well.